db/inst/import-try/01.global.subset.R

```diff
@@ -0,0 +1,36 @@
+# 1. Global subset
+library(data.table)
+library(bit64)
+message("Loading TRY data...")
```

Comment:
not sure if it is necessary, but I just want to point out that @robkooper developed the `logger.*` functions for printing information to the console in a standard, globally controllable way